composer.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "talis/persona-php-client",
   "description": "This is a php client library for Persona supporting generation, validation and caching of Oauth Tokens",
-  "version": "4.0.1",
+  "version": "4.0.2",
```

Comment:
This is a significant change of behaviour, that I think deserves a minor, rather than patch bump.